pandas/core/internals/managers.py

```diff
@@ -719,7 +719,6 @@ def get_numeric_data(self, copy: bool = False) -> "BlockManager":
         copy : bool, default False
             Whether to copy the blocks
         """
-        self._consolidate_inplace()
```

Comment:
if you are removing this then get_bool_data right above looks suspicious.

Reply:
i agree.  thats part of the _reduce trouble we discussed this afternoon.  #34918 removes that usage.